docs/Configuration.md

```diff
@@ -291,13 +320,15 @@ batch pool of tests, but is used in combination of
 
 ### lock_on_queue
 
-An integer.
+A non-negative integer. Max length 9 digits.
+
 A label to associate a test to a specific Test Agent.
 Default value: `0`.
```

Comment:
This is just a name, isn't it? Or will that create that many queues? The description is incomplete. And the maximum value is probably unnecssarily hign.